austrakka/components/field/__init__.py

```diff
@@ -24,51 +24,54 @@ def field_list(out_format: str):
 @field.command('add', hidden=hide_admin_cmds())
 @opt_name(help="Field name")
 @opt_fieldtype()
-@click.option('-d', '--description', 'description',
-              help="This field describes the purpose of the metadata field. "
-                   "Its value is also used for generating XLSX pro forma files.")
-@click.option('--nndss', 'nndss',
-              help="The corresponding National Notifiable Diseases Surveillance System label, "
-                   "where it exists.")
-@click.option('--colour-nodes', 'colour_nodes', flag_value='viz',
-              help="This field may be used to colour nodes on the tree")
-@click.option('--no-colour-nodes', 'colour_nodes', flag_value='no_viz',
-              help="This field may not be used to colour nodes on the tree")
-@click.option('-O', '--column-order', type=int, default=9000,
-              help="Default order in which this column will be sorted in tables relative to other '"
-                   "fields. If no value is specifed, the column will be placed after ordered "
+@create_option('-d', '--description', 'description',
```

Comment:
Technically these description fields should use the existing opt_description (and pass in help). All the rest look good